src/pages/Improbability/Improbability.component.tsx

```diff
@@ -0,0 +1,69 @@
+import Typography from '@mui/material/Typography';
+import { useForm } from 'react-hook-form';
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import { DataTable } from '../../components/DataTable/DataTable.component';
+import { PageView } from '../../components/PageView/PageView.component';
+import {
+  ImprobabilityComponentProps,
+  ImprobabilityFormValues,
+  ImprobabilityRowData,
+} from './Improbability.types';
+import { ControlledDropdown } from '../../components/Dropdown/Dropdown.controlled';
+import {
+  coinsListColumns,
+  improbabilityDropdownOptions,
+} from './Improbability.utils';
+
+export const ImprobabilityComponent = ({
+  state,
+  coins,
+}: ImprobabilityComponentProps) => {
+  const { setValue, control } = useForm<ImprobabilityFormValues>({
+    mode: 'onChange',
+    defaultValues: { network: '' },
```

Comment:
I know it's a really small form, but I would suggest adding a `.fields` file with enum with field names and defaultValues to make it more consistent